websites/V/Vizzy/presence.ts

```diff
@@ -0,0 +1,122 @@
+const presence = new Presence({
+		clientId: "968353669491871754"
+	}),
+	browsingTimestamp = Math.floor(Date.now());
+function getElementByXpath(path: string) {
+	return document.evaluate(
+		path,
+		document,
+		null,
+		XPathResult.FIRST_ORDERED_NODE_TYPE,
+		null
+	).singleNodeValue;
+}
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			smallImageKey: "logo_512",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, search } = window.location;
+	switch (true) {
```

Comment:
all you do in this switch statement is compare a variable to a string so please do a proper switch statement with the variable in here and the cases down below